src/less/select/ds4/select.less

```diff
@@ -0,0 +1,74 @@
+@import "../../less/ds4/variables.less";
+@import "../../less/ds4/mixins.less";
+
+@import "../base/select.less";
+
+.select__icon:empty {
```

Comment:
Hmm, we're using `span` element selector everywhere else rather than `:empty` for the background SVG. Reason being that we keep the specificity the same when we need to add any `svg` element rules. Not sure if `:not(:empty)` would have the same specificity but I think the element selector is more readable. Maybe we should stick with consistency for now?

Reply:
Can you show me an example of what you mean?

But, yes, I want to make sure it's consistent.

Reply:
```css
// base
.select__icon {
  ..
}
// background specific
span.select__icon {
  ...
}
// foreground specific
svg.select__icon {
  ...
}
```

Rather than

```css
// base
.select__icon {
  ..
}
// background specific
.select__icon:empty {
  ...
}
// foreground specific
svg.select__icon {
  ...
}
```

Just to be consistent with elsewhere.